docs/editors/vim.md

```diff
@@ -89,15 +83,13 @@ Here's a recommended configuration:
 " ~/.vimrc
 " Configuration for coc.nvim
 
+" If hidden is not set, TextEdit might fail.
 set hidden
 
 " Some servers have issues with backup files
 set nobackup
 set nowritebackup
 
-" Better display for messages
-set cmdheight=2
```

Comment:
I've started to recommend people not set this especially since the status bar integration.